onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -531,10 +531,11 @@ static bool IsUnsupportedOpMode(const onnxruntime::GraphViewer& graph_viewer, co
 
     const auto& attributes = node->GetAttributes();
     if (attributes.count("starts") > 0 && attributes.count("ends") > 0) {
-      auto starts = toVector((*attributes.find("starts")).second.ints());
-      auto ends = toVector((*attributes.find("ends")).second.ints());
+      const ONNX_NAMESPACE::int64s& starts = (*attributes.find("starts")).second.ints();
+      const ONNX_NAMESPACE::int64s& ends = (*attributes.find("ends")).second.ints();
+
       for (std::size_t i = 0; i < starts.size(); ++i) {
-        if (starts.at(i) > ends.at(i)) {
```

Comment:
Removing .at() removes bounds checking here why is this being changed? https://en.cppreference.com/w/cpp/container/vector/at.html

If start/end is set to -1 that implies there's something else broken here or needs support.